From: Jakub Steiner <jimmac@gmail.com>
Date: Sat, 5 Aug 2023 20:35:48 +0200
Subject: theme: Make today weekend fg color legible

- when a weekend is today, the insensitive color was illegible

Fixes https://gitlab.gnome.org/GNOME/gnome-shell/-/issues/6846

Part-of: <https://gitlab.gnome.org/GNOME/gnome-shell/-/merge_requests/2857>
(cherry picked from commit b6658cd2b7e7da943fecbd73416659e7c073fae4)
---
 data/theme/gnome-shell-sass/widgets/_calendar.scss | 1 +
 1 file changed, 1 insertion(+)

diff --git a/data/theme/gnome-shell-sass/widgets/_calendar.scss b/data/theme/gnome-shell-sass/widgets/_calendar.scss
index 0cabd59..fc054da 100644
--- a/data/theme/gnome-shell-sass/widgets/_calendar.scss
+++ b/data/theme/gnome-shell-sass/widgets/_calendar.scss
@@ -98,6 +98,7 @@
 
     &.calendar-today {
       @extend .button, .default;
+      color: $selected_fg_color; /* override weekends */
     }
 
     &.calendar-day-with-events {
